package com.otmanethedev.beers.data.data_source.errors

object UnknownException : Throwable("Unknown error occurred")